src/map/mob.cpp

```diff
@@ -1267,18 +1267,23 @@ static int32 mob_can_changetarget(struct mob_data* md, struct block_list* target
  * Randomizes the target ID of a monster if it has the given mode
  * @param bl Unit that is going to attack
  * @param target_id Target ID to modify
+ * @return Whether a target was found (true) or not (false)
  */
-void mob_randomtarget(mob_data& md, int32& target_id) {
+bool mob_randomtarget(mob_data& md, int32& target_id) {
 	if (!status_has_mode(&md.status, MD_RANDOMTARGET))
-		return;
+		return true;
```

Comment:
True means that a target was found according to your doc?
May need a little extra comment, why it is different in this case.

Reply:
Well, the target was found, it's the current target for monsters without random target mode. But I guess I can add that as comment.
```suggestion
		return true; // Keep current target
```
Like this.